org.gnome.Keysign.yml

```diff
@@ -0,0 +1,306 @@
+id: org.gnome.Keysign
+runtime: org.gnome.Platform
+runtime-version: '3.28'
+sdk: org.gnome.Sdk
+command: gnome-keysign
+copy-icon: true
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --system-talk-name=org.freedesktop.Avahi
+  - --device=all
+  - --filesystem=~/.gnupg:ro
+  - --filesystem=xdg-run/gnupg:ro
+# We're waiting for webcam support: https://github.com/flatpak/xdg-desktop-portal/issues/38;
+# we use --device=all meanwhile. We do network, because we're opening a port.
+appstream-compose: false
+build-options:
+  cflags: -O2 -g
+  cxxflags: -O2 -g
+  env:
+    V: '1'
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /share/pkgconfig
+  - /share/aclocal
+  - /man
+  - /share/man
+  - /share/gtk-doc
+  - /share/vala
+  - "*.la"
+  - "*.a"
+modules:
+  - name: setuptools
+    buildsystem: simple
+    ensure-writable:
+      - easy-install.pth
+      - setuptools.pth
+    build-commands:
+      - python2 setup.py build
+      - python2 setup.py install --prefix=${FLATPAK_DEST}
+    sources:
+      - type: archive
+        url: https://pypi.python.org/packages/32/3c/e853a68b703f347f5ed86585c2dd2828a83252e1216c1201fa6f81270578/setuptools-26.1.1.tar.gz#md5=0744ee90ad266fb117d59f94334185d0
+        sha256: 475ce28993d7cb75335942525b9fac79f7431a7f6e8a0079c0f2680641379481
+
+  - name: pip
```

Comment:
Do you need `pip` at runtime? If not its already in the Sdk.

Reply:
No I don't but the pip2 version inside the Sdk is the old 7.1.0 and it doesn't allow the usage of `--prefix`.
You can easily try it with this manifest https://ptpb.pw/Nuee

Installing a newer pip version solved that problem.